### PR TITLE
update hardhat docs for ethers v6

### PR DESCRIPTION
**Pull request by eshaben**
### Description

Hardhat now supports Ethers v6, so this PR updates the hardhat docs to support v6.

### Checklist

- [x] If this requires translations for the `moonbeam-docs-cn` repo, I have created a ticket for the translations in Jira

### After Translation Requirements

- [x] Will need to create PR in `moonbeam-docs` repo to remove images

#### Items to be Updated

`/images/tutorials/eth-api/hardhat-start-to-end/hardhat-2.png` will need to be removed and the rest of the hardhat images will need to be reordered

### CODE REVIEW

File: tutorials/eth-api/hardhat-start-to-end.md [hunk 1]
```diff
@@ -102,66 +115,36 @@ To get started, take the following steps:
...
+5. Add your [Moonscan API key](/builders/build/eth-api/verify-contracts/etherscan-plugins/#generating-a-moonscan-api-key){target=_blank}, which is required for the verification steps we'll be taking later in this tutorial
 
 ```js
 // 1. Import the Ethers, Hardhat Toolbox, and Etherscan plugins 
 // required to interact with our contracts
-require("@nomicfoundation/hardhat-toolbox");
-require("@nomiclabs/hardhat-etherscan");
+require('@nomicfoundation/hardhat-toolbox');
+require('@nomiclabs/hardhat-etherscan');
 require('@nomiclabs/hardhat-ethers');
```
Comment (themacexpert) [inline tutorials/eth-api/hardhat-start-to-end.md:137]:
```suggestion
require('@nomicfoundation/hardhat-ethers');
```

**Review by themacexpert — APPROVED** [review]
I ran thru the builders/dev-env hardhat page and it worked perfectly 
LGTM

**Comment by eshaben** [comment]
@themacexpert would you mind taking a look at my last couple of commits? sorry, I realized I should update all use cases of nomiclabs to nomicfoundation

**Review by themacexpert — APPROVED** [review]
Updates look good 👍

**Pull request merged**